packages/maskbook/src/plugins/ITO/hooks/useClaimCallback.ts

```diff
@@ -1,41 +1,40 @@
 import { useCallback } from 'react'
 import type { TransactionReceipt } from 'web3-core'
 import type { Tx } from '@dimensiondev/contracts/types/types'
-import { TransactionStateType, useTransactionState } from '../../../web3/hooks/useTransactionState'
 import { useAccount } from '../../../web3/hooks/useAccount'
 import { TransactionEventType } from '../../../web3/types'
 import { addGasMargin } from '../../../web3/helpers'
+import { TransactionStateType, useTransactionState } from '../../../web3/hooks/useTransactionState'
 import { useChainId } from '../../../web3/hooks/useChainState'
-import { useMaskITO_Contract } from '../contracts/useMaskITO_Contract'
+import { useITO_Contract } from '../contracts/useITO_Contract'
```

Comment:
So from now on, users of Mask ITO are unable to claim locked tokens by Mask extension?